### ISSUE TITLE
input_tensor option for segmentation model

### ISSUE THREAD

**Issue by cwchengtw** [qubvel/segmentation_models#182]
I have the need to pass an input_tensor to the segmentation model just like I do in keras. application. I see your code have **kwargs in all segmentation model and can be pass to backbone model.
But when I do this, I get an error from the get_submodules_from_kwargs(kwargs).
Because get_submodules_from_kwargs just accept "backend, layers, models, keras_utils"  kwargs，it result an error.
I suggest to add default input_tensor=None to all segmentation model and it will work.

**Comment by qubvel** [comment]
Hi @cwchengtw  
It might be an option, but not sure you will be able to load `encoder_weights` in this case.

**Comment by cwchengtw** [comment]
Hi @qubvel

I used to train model in keras application and classification model, and deploy model  in tensorflow saved model or frozen model format. Before I convert from keras model to tensorflow model, I add a
tf.placeholder and some ops to do image transformation (mean, or normalize) and redefine keras model with transform input tensor and load the training weight.
After doing that, I convert model to tensorflow saved model or frozen model with input_tensor and model.outputs, I will not to do image preprocessing in deployment.

I tried encoder_weights and model.load_weights in this case, it work. (in keras)

And, I have another issue with tfkeras in classification model (not an issue in tensorflow keras application). When I used input_tensor in tfkeras, backend.is_keras_tensor will not work, and it raise an error. (I think tensorflow.keras.application really use keras backend in this case.)
